packages/components/src/slot-fill/stories/index.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { createContext, useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Slot, Fill, Provider as SlotFillProvider } from '../';
+
+const meta: ComponentMeta< typeof Slot > = {
+	component: Slot,
+	title: 'Components/SlotFill',
+	subcomponents: { Fill, SlotFillProvider },
+	argTypes: {
+		name: { control: { type: null } },
+		as: { control: { type: 'text' } },
+		fillProps: { control: { type: null } },
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
+};
+
+export default meta;
+
+export const Default: ComponentStory< typeof Slot > = ( props ) => {
+	return (
+		<SlotFillProvider>
+			<h2>Profile</h2>
+			<p>
+				Name: <Slot { ...props } name="name" />
+			</p>
+			<p>
+				Age: <Slot { ...props } name="age" />
+			</p>
+			<Fill name="name">Grace</Fill>
+			<Fill name="age">33</Fill>
+		</SlotFillProvider>
+	);
+};
+Default.args = {
+	className: 'my-slot',
```

Comment:
https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/slot-fill/README.md#props

Normally, I believe that className is unnecessary, but in this case, whether or not className is accepted depends on bubblesVirtually, so I thought it would be better to make it visible.

Reply:
Just wanted to flag that at the moment I don't see this information reflected in the `types` — TypeScript is allowing me to pass a `className` even when `bubblesVirtually` is `false`

Reply:
@ciampo 
Enabled className only if bubblesVirtually:true.
https://github.com/WordPress/gutenberg/pull/51350/files#diff-7102b9f3a6fad3b8e9a616948e5c757e6982548b299628fa8ab170aee7399302R37-R47